scripts/update_nolints.sh

```diff
@@ -9,9 +9,9 @@ owner_name=leanprover-community
 git fetch $remote_name
 git rev-parse --verify --quiet refs/remotes/$remote_name/$branch_name && exit 0
 
-# Exit if there are no changes with respect to master
+# Exit if there are no changes relative to master
 git diff-index --quiet refs/remotes/$remote_name/master -- scripts/nolints.txt && exit 0
-# Exit if staging exists and there are no changes with respect to staging
+# Exit if staging exists and there are no changes relative to staging
```

Comment:
The staging test is no longer necessary with the nightly cronjob, right?

Reply:
Yeah, I'll remove it.